README.md

```diff
@@ -134,7 +138,7 @@ This approach reduces the administrative power of the Ansible host and prevents
 Moving secret retrieval to the node provides a maximum level of security. This approach reduces the security risk by
 providing each node with the minimal amount of privilege required for that node. The Conjur Module also provides host level audit logging of secret retrieval. Environment variables are never written to disk.
 
-The module receives variables and a command as arguments and, as in [Conjur's Summon CLI](https://www.conjur.org/integrations/summon.html),
+The module receives a secretsyml string (YAML format) and a command as arguments and, as in [Conjur's Summon CLI](https://www.conjur.org/integrations/summon.html),
```

Comment:
should this be `secrets.yml` instead of `secretsyml`?

Reply:
ah, I see below.  How about just `secrets` rather than `secretsyml`?  Technically Ansible config is YAML. It feels weird to re-declare it as type YAML.

Reply:
it'd have to specifically be a string. if it's part of the YAML of the ansible config then we're unable to parse it downstream. we'd only get the crums from ansible having parse it first. e.g. 
```
secretsyml:
   DB_PASSWORD: !var /dev/db password
```
will throw an error as part of a playbook. that's why i shifted to string and felt the need to reiterate that the format of the string is YAML. interestingly with the string we're able to do this:

```
secretsyml: {{lookup('file', '/file/path/to/secrets/yml')}}
```